snorkel/labeling/analysis.py

```diff
@@ -11,87 +11,123 @@
+    """Get indicator vector z where z_i = 1 if x_i is labeled by more than one LF."""
     return np.where(np.ravel((L != 0).sum(axis=1)) > 1, 1, 0)
 
 
 def _conflicted_data_points(L: sparse.spmatrix) -> np.ndarray:
-    """Returns an indicator vector where ith element = 1 if x_i is labeled by
-    at least two LFs that give it disagreeing labels."""
+    """Get indicator vector z where z_i = 1 if x_i is labeled differently by two LFs."""
     m = sparse.diags(np.ravel(L.max(axis=1).todense()))
     return np.ravel(np.max(m @ (L != 0) != L, axis=1).astype(int).todense())
 
 
 def label_coverage(L: Matrix) -> float:
-    """Returns the **fraction of data points with > 0 (non-zero) labels**
-    Args:
-        L: an n x m scipy.sparse matrix where L_{i,j} is the label given by the
-            jth LF to the ith item
+    """Compute the fraction of data points with at least one label.
+
+    Parameters
+    ----------
+    L
+        Matrix where L_{i,j} is the label given by the jth LF to the ith candidate
+
+    Returns
+    -------
+    float
+        Fraction of data points with labels
     """
     return _covered_data_points(L).sum() / L.shape[0]
 
 
 def label_overlap(L: Matrix) -> float:
-    """Returns the **fraction of data points with > 1 (non-zero) labels**
-    Args:
-        L: an n x m scipy.sparse matrix where L_{i,j} is the label given by the
-            jth LF to the ith item
+    """Compute the fraction of data points with at least two (non-abstain) labels.
+
+    Parameters
+    ----------
+    L
+        Matrix where L_{i,j} is the label given by the jth LF to the ith candidate
+
+    Returns
+    -------
+    float
+        Fraction of data points with overlapping labels
     """
     return _overlapped_data_points(L).sum() / L.shape[0]
 
 
 def label_conflict(L: sparse.spmatrix) -> float:
-    """Returns the **fraction of data points with conflicting (disagreeing)
-    labels.**
-    Args:
-        L: an n x m scipy.sparse matrix where L_{i,j} is the label given by the
-            jth LF to the ith item
+    """Compute the fraction of data points with conflicting (non-abstain) labels.
+
+    Parameters
+    ----------
+    L
+        Matrix where L_{i,j} is the label given by the jth LF to the ith candidate
+
+    Returns
+    -------
+    float
+        Fraction of data points with conflicting labels
     """
     return _conflicted_data_points(L).sum() / L.shape[0]
 
 
 def lf_polarities(L: Matrix) -> List[List[int]]:
-    """Return the polarities of each LF based on evidence in a label matrix.
+    """Infer the polarities of each LF based on evidence in a label matrix.
 
-    Args:
-        L: an n x m scipy.sparse matrix where L_{i,j} is the label given by the
-            jth LF to the ith candidate
+    Parameters
+    ----------
+    L
+        Matrix where L_{i,j} is the label given by the jth LF to the ith candidate
+
+    Returns
+    -------
+    List[List[int]]
+        Unique output labels for each LF
```

Comment:
Example- again I just worry that these all are not quite intuitive from our arbitrary internal naming alone, and should be crystal clear here